mozilla_bitbar_devicepool/test_run_manager.py

```diff
@@ -60,7 +60,10 @@ def get_bitbar_test_stats(self, project_name, project_config):
         device_group_count = bitbar_device_group['deviceCount']
 
         offline_devices = []
-        temp_offline_devices = get_offline_devices(device_model=project_config['device_model'])
+        if 'device_model' in project_config:
+            temp_offline_devices = get_offline_devices(device_model=project_config['device_model'])
+        else:
+            temp_offline_devices = get_offline_devices()
```

Comment:
This is ok but we could do this
I understand device_model doesn't make that much sense for your test projects.

```
temp_offline_devices = get_offline_devices(device_model=project_config.get('device_model', None))
```

Reply:
Changed to your suggestion.